include/OrgasmControl.h

```diff
@@ -60,11 +67,24 @@ namespace OrgasmControl {
     // File Writer
     long recording_start_ms = 0;
     File logfile;
+    
+    //  Post Orgasm Clench variables
+    long clench_pressure_threshold = 4096;
+    int clench_duration = 0;
 
+    // Autoedging Time and Post-Orgasm varables
+    long auto_edging_start_millis;
+    long post_orgasm_start_millis;
+    long post_orgasm_duration_millis;
+    bool menu_is_locked = false;
+    bool detected_orgasm = false;
+    int post_orgasm_duration_seconds = Config.post_orgasm_duration_seconds;
```

Comment:
These variables are initialized at compile time, right? So this will pick up whatever the default value is of that config option from its global initialization, which at this point might be an uninitialized value on its own.

If you need to initialize this to the current configuration value, you will want to find a way to do it after the config has been loaded from the SD card.

Reply:
ok i'll look into this issue

Reply:
ok, fixed this, must have forgotten to remove because i had integrated in the Config.ccp